app/components/markdown/markdown_image/index.tsx

```diff
@@ -207,58 +218,66 @@ const MarkdownImage = ({
                     {' '}
                 </Text>
             );
+        } else if (fileInfo.extension === 'svg') {
+            image = (
+                <SvgUri
+                    uri={fileInfo.uri!}
+                    style={{flex: 1, backgroundColor: changeOpacity(theme.centerChannelColor, 0.06), borderRadius: 8}}
+                    width={fileInfo.width || Math.min(width, height)}
```

Comment:
Same question about having a maximum size.

Reply:
same answer.. btw SVG's are vector and if the user is not specifying the size we are assigning the size based on the viewport as the metadata does not contain any size information about svg's

Reply:
actually I need to change this to

```
width={Math.min(fileInfo.width, width, height)}
height={Math.min(fileInfo.height, width, height)}
```

then is all fine.